config/prod.exs

```diff
@@ -52,13 +52,6 @@ config :mindwendel, MindwendelWeb.Endpoint,
 # Do not print debug messages in production
 config :logger, level: :info
 
-config :mindwendel, :options,
-  feature_brainstorming_teasers:
```

Comment:
this was doubled up, exists in `runtime.exs` just like the others (as it should be imo)

Reply:
good catch, thanks